recipes/pycontour/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is already included in the PyPI tarball, so you can delete it from this PR.

Reply:
Done.